ci/docker/base.dockerfile

```diff
@@ -43,8 +43,7 @@ ENV CC=$CC \
 
 ENV CCACHE_COMPILERCHECK=content \
     CCACHE_COMPRESS=1  \
-    CCACHE_COMPRESSLEVE=5  \
```

Comment:
There's a typo (`CCACHE_COMPRESSLEVEL`), but also default value is 6 so let's simply stick with default.